eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxow.hs

```diff
@@ -487,7 +487,7 @@ witsVKeyNeeded utxo' tx genDelegs =
     updateKeys =
       asWitness
         `Set.map` propWits
-          (strictMaybeToMaybe $ txBody ^. updateTxBodyL)
+          (strictMaybeToMaybe $ txBody ^. updateTxBodyG)
```

Comment:
so we can drop the `ProtVerAtMost era 8` constraint since we now only need a getter? nice!